PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,51 @@
+#### Name
+
+Mac - Set JumpCloud Password Manager Release Channel | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+#!/bin/bash
+
+# Set releaseChannel to beta OR dogfood OR public depending on your desired release channel
+releaseChannel="dogfood"
```

Comment:
Let's change this to public so that every script is set to the public channel by default? 
```suggestion
releaseChannel="public"
```